config/initializers/klarna_service.rb

```diff
@@ -0,0 +1,17 @@
+KLARNA_CHECKOUT_URL = ENV['SHF_KLARNA_CHECKOUT_URL']
+KLARNA_ORDER_MGMT_URL = ENV['SHF_KLARNA_ORDER_MGMT_URL']
+
+if Rails.env.production?
+  KLARNA_API_AUTH_USERNAME  = ENV['SHF_KLARNA_API_AUTH_USERNAME']
+  KLARNA_API_AUTH_PASSWORD  = ENV['SHF_KLARNA_API_AUTH_PASSWORD']
+else
+  KLARNA_API_AUTH_USERNAME  = ENV['SHF_KLARNA_API_AUTH_USERNAME_DEV']
+  KLARNA_API_AUTH_PASSWORD  = ENV['SHF_KLARNA_API_AUTH_PASSWORD_DEV']
+end
+
+SHF_MEMBER_FEE   = 30000  # unit == one hundredth krona
```

Comment:
We should really pull these 2 fees out into ENV files.
They shouldn't be buried in gem initialization files (either HIPS or KLARNA).
Can do this now or in a later PR.

(Just skimming thru your changes.)

Reply:
Not sure I agree.   These values are 1) relatively invariant (unless SHF changes fees every few years, but certainly not very often), and 2) there is no reason to use different  values in different environments.  The overhead of maintaining ENV vars, in different environments, is greater than the convenience of having them dynamically settable.

Reply:
Klarna does not have anything to do with setting these values.  Klarna is not responsible for these values in any way.  Thus they should not be associated with Klarna or the Klarna API (or HIPS for that matter.)

If they are "relatively invariant" then how can it also be important to be "dynamically settable"?  

The fees are really the responsibility of the admin, just like other values that we have in the Admin Config.  (We also have some values in the ENV files that should really be in the Admin config.)